nightlies_watcher/config.py

```diff
@@ -0,0 +1,39 @@
+import json
+import logging
+import os
+from frozendict import frozendict
+
+from nightlies_watcher.directories import PROJECT_DIRECTORY, DATA_DIRECTORY
+
+logger = logging.getLogger(__name__)
+
+_configs = {}
+
+DEFAULT_LOCATION = os.path.join(PROJECT_DIRECTORY, 'config.json')
+DEFAULT_CONFIG_LOCATION = os.path.join(DATA_DIRECTORY, 'config.default.json')
+
+
+def get_config(config_path=None):
+    config_path = config_path or DEFAULT_LOCATION
+
+    try:
+        return _configs[config_path]
+    except KeyError:
```

Comment:
This looks like a perfect example to use [ChainMap](https://docs.python.org/3/library/collections.html#collections.ChainMap)!